components/code-editor/editor.js

```diff
@@ -99,12 +99,12 @@ class CodeEditor extends Component {
 		}
 
 		if ( ! this.props.focus && this.editor.hasFocus() ) {
-			document.activeElement.blur();
+			document.activeElement.change();
```

Comment:
`HTMLElement` doesn't have a `change()` method, so this would always result in an `undefined is not a function` exception.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement